Mathlib/Data/Sym/Basic.lean

```diff
@@ -54,6 +54,9 @@ instance Sym.hasCoe (α : Type _) (n : ℕ) : CoeOut (Sym α n) (Multiset α) :=
   ⟨toMultiset⟩
 #align sym.has_coe Sym.hasCoe
 
+-- Porting note: instance needed for Data.Finset.Sym
+instance [DecidableEq α]: DecidableEq (Sym α n) := Subtype.instDecidableEqSubtype
```

Comment:
This instance appears twice in this PR

Reply:
I removed the extra one in Data.Sym.Sym2